test/postman_test/SSAS.postman_collection.json

```diff
@@ -22,6 +22,21 @@
 				}
 			],
 			"request": {
+				"auth": {
+					"type": "basic",
+					"basic": [
+						{
+							"key": "password",
+							"value": "{{adminClientSecret}}",
```

Comment:
Can this be specified at the collection level and inherited?  The Postman app implies this might be possible, and it would add less noise to the collection.

Also, `test/postman_test/SSAS_Smoke_Test.postman_collection.json` will need the same treatment as this file.